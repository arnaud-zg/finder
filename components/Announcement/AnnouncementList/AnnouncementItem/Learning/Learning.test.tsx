import React from 'react'
import renderer from 'react-test-renderer'
import { Learning } from '.'

describe('Learning', () => {
  it('renders correctly', () => {
    const testRenderer = renderer.create(
      <Learning
        courseDuration="2h"
        description="Improve your Online Shopify Store with France's Top Shopify Expert"
        identifier="2"
        price={9700}
        publishTime="2019-09-29T13:50:42.926Z"
      />
    )
    expect(testRenderer.toJSON()).toMatchSnapshot()
  })
})
