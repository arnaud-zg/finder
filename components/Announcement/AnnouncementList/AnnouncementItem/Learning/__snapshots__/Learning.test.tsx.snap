// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Learning renders correctly 1`] = `
<div
  id="2"
>
  <h3
    className="fiird0z"
  >
    Improve your Online Shopify Store with France's Top Shopify Expert
  </h3>
  <div
    className="f18ec881"
  >
    2h
  </div>
  <div
    className="f18ec881"
  >
    97
     â‚¬
  </div>
  <div
    className="f18ec881"
  >
    29/09/2019 at 13:50
  </div>
</div>
`;
